portfolio/src/main/java/com/google/sps/data/EntityCreation.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+// TODO: add javadoc
+public interface EntityCreation<T> {
```

Comment:
Nit: Maybe EntityConvertibleCreator? to be symmetric with the above interface?

Reply:
Fixed